SecretSanta/tests/SecretSanta.Api.Tests/Controllers/UsersControllerTests.cs

```diff
@@ -0,0 +1,112 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using System.Collections.Generic;
+using SecretSanta.Api.Controllers;
+using SecretSanta.Business;
+using SecretSanta.Data;
+
+namespace SecretSanta.Api.Tests
+{
+    [TestClass]
+    public class UsersControllerTests
+    {
+        [TestMethod]
+        public void GetList_WhenCalled_ReturnsListofUsers()
+        {
+            string expectedString = "Actually Called";
+            Mock<IUserRepository> mock = new();
+            mock.Setup(item => item.List())
+            .Returns(new List<User>()
+            {
+                new User()
+                {
+                    FirstName = expectedString,
+                    LastName = "Doesn't Matter",
+                    Id = 0
+                }
+            });
+            UsersController usersController = new(mock.Object);
+
+            Assert.AreEqual<string>(usersController.Get()[0].FirstName, expectedString);
```

Comment:
Nit: The parameters for all of the `Assert` methods go in the order <expected value> <actual value>. The order appears backwards here. Not a big deal but it affect the failure message that is displayed when the values are not equal.
```suggestion
            Assert.AreEqual<string>(expectedString, usersController.Get()[0].FirstName);
```